src/SqlPersistence.PersistenceTests/PersistenceTestsConfiguration.cs

```diff
@@ -0,0 +1,171 @@
+﻿namespace NServiceBus.PersistenceTesting
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Data.Common;
+    using System.Threading.Tasks;
+    using Extensibility;
+    using Newtonsoft.Json;
+    using Npgsql;
+    using NpgsqlTypes;
+    using NServiceBus.Outbox;
+    using NServiceBus.Sagas;
+    using NUnit.Framework;
+    using Persistence;
+    using Persistence.Sql.ScriptBuilder;
+    using Transport;
+
+    public partial class PersistenceTestsConfiguration
+    {
+        public bool SupportsDtc => false; // TODO: this should actually be set to true, tests still failing
```

Comment:
What do we need to do with this TODO?

Reply:
Actually, now that I see it, I need to add this to the ignored tests issue and the todo can be removed